doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -645,6 +641,9 @@ which reduction engine to use.)  For example:
             Goal 1 + 0 = 1.
             unfold Nat.add.
 
+      .. exn:: Cannot coerce @qualid to an evaluable reference.
```

Comment:
This error cannot be raised anymore by this tactic. It can though be raised from the `Hint Transparent` command, as noticed by Jason, so I would move this `exn::` there.

Reply:
Moved it, but in general, matching error messages to specific commands/tactics doesn't make a huge amount of sense (not 1-1, hard to maintain, etc.)

Reply:
Agreed. We will have to think of a better way.